package.json

```diff
@@ -27,16 +27,17 @@
     "watch": "npm run watch:all",
```

Comment:
you need to update the `build:site` task to remove the `mini-book-site-generator` reference:

```diff
-"antora --generator=mini-book-site-generator --html-url-extension-style=indexify site.yml
+"antora --html-url-extension-style=indexify site.yml
```